tests/fixtures.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+
+from bitshares import BitShares
+from bitshares.instance import set_shared_blockchain_instance
+from bitshares.account import Account
+# default wifs key for testing
+wifs = [
+    "5J6r6kzPcKyrhdUGBSxA95spFokg3uj82k5Npj54Dfc4TcACeY8"
```

Comment:
Should not put a WIF in the code. Can we make this an environment variable?

Reply:
it's test wif,i think it's ok.

Reply:
but,if you insist,i will get rid of it.

Reply:
Better to not have account names, passwords or keys in repo, even if they are for testing purpose only.

Reply:
it done

Reply:
Test files naming convention is test_xxx_yyy.py

Reply:
Ok, pytest supports both `test_*.py` or `*_test.py`, we need to pick the single naming. I personally prefer 1st

Reply:
@thehapax I opened #572 for that, let's do a choice there.